docs/document/docusaurus.config.js

```diff
@@ -0,0 +1,163 @@
+          editUrl: undefined,
+          lastVersion: `${getLatestVersionName()}`,
+        },
+        theme: {
+          customCss: [
+            require.resolve('./src/css/custom.css'),
+            require.resolve('./src/css/kuc-custom.css')
+          ],
+        },
+      }),
+    ],
+  ],
+  themeConfig:
+    /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
+    ({
+      navbar: {
+        title: '',
+        logo: {
+          alt: 'kintone UI Component',
+          src: 'img/kuc_logo_white.png',
+        },
+        items: [
+          {
+            type: 'docsVersion',
+            position: 'left',
+            to: '/versions',
+            className: 'versions'
+          },
+          {
+            type: 'doc',
+            docId: 'getting-started/quick-start',
+            position: 'right',
+            label: 'Documentation',
+          },
+          {
+            type: 'doc',
+            docId: 'releases/release-notes',
+            position: 'right',
+            label: 'Releases',
+          },
+          {
+            href: 'https://github.com/kintone-labs/kintone-ui-component',
+            label: 'GitHub',
+            position: 'right',
+          },
+          {
+            type: 'localeDropdown',
+            position: 'right',
+          },
+          {
+            type: 'search',
+            position: 'right',
+          }
+        ],
+      },
+      footer: {
+        style: 'dark',
+        links: [
+          {
+            title: 'Documentation',
+            items: [
+              {
+                label: 'Quick Start',
+                to: '/docs/getting-started/quick-start',
+              },
+              {
+                label: 'Components',
+                to: '/docs/components/desktop/attachment',
+              },
+            ],
+          },
+          {
+            title: 'Community',
+            items: [
+              {
+                label: 'GitHub issue',
+                href: 'https://github.com/kintone-labs/kintone-ui-component/issues',
+              }
+            ],
+          },
+          {
+            title: 'Other',
+            items: [
+              {
+                label: 'Privacy Policy',
+                href: 'https://cybozu.co.jp/privacy/',
+              },
+            ],
+          },
+        ],
+        copyright: `Copyright © ${new Date().getFullYear()} Cybozu Inc.`,
+      },
+      prism: {
+        theme: lightCodeTheme,
+        darkTheme: darkCodeTheme,
+      },
+      colorMode: {
+        defaultMode: 'light',
+        disableSwitch: true,
+        respectPrefersColorScheme: false,
+      },
+      algolia: {
+        appId: 'QJ2QB6FCE5',
+        apiKey: '636c5f8d218f77cc2f87c81fbd95a9b8',
+        indexName: 'kuc-usaurus-v2',
+        contextualSearch: true,
+      },
+    }),
+  plugins: [
+    [
+      '@docusaurus/plugin-google-tag-manager',
+      {
+        containerId: 'GTM-5N49D3',
+      },
```

Comment:
When we use v2, we can use this plugin instead of this code : https://github.com/kintone-labs/kintone-ui-component/blob/master/docs/document/website/static/js/gtmHeader.js